gripper_action_controller:
  # Feedback rate in Hz
  publish_rate: 100
  min_gap_counts: 224
  # Gripper width in meters [0, 0.085]
  min_gap:    0.0
  max_gap:    0.085
  # Gripper speed in m/s [0.013, 0.1]
  min_speed:  0.013
  max_speed:  0.1
  # Gripper force in N. [40, 100]
  min_force:  40.0
  max_force:  100.0
